web/src/main/java/com/marklogic/hub/web/web/NewJobController.java

```diff
@@ -20,18 +21,28 @@
     @RequestMapping(method = RequestMethod.GET)
     @ResponseBody
     public ResponseEntity<?> getJobs(@RequestParam(value = "flowName", required = false) String flowName) throws IOException {
-        jobService.setupClient();
-        List<JobModel> jobModels = jobService.getJobs(flowName);
-        jobService.release();
+        List<JobModel> jobModels;
+        try {
+            jobService.setupClient();
+            jobModels = jobService.getJobs(flowName);
+            jobService.release();
```

Comment:
I know @ehennum has mentioned before that there's some overhead with construction of a DatabaseClient, and since it's a thread-safe object, it's ideal for performance if it can be instantiated once and used many times. 

Since NewJobService is a singleton object managed by Spring, is there any risk to having it instantiate a new Jobs DatabaseClient in a PostConstruct-annotated method on startup? The only risk I can see is if the config in HubConfigImpl changed... perhaps that's possible when a new project is selected? Though ideally, that would refresh the Spring container with the new config.